llama.cpp

```diff
@@ -948,7 +950,22 @@ struct llama_hparams {
     float rope_freq_scale_train;
 
     bool operator!=(const llama_hparams & other) const {
```

Comment:
It might be more intuitive to change this to `operator==` and swap false and true return values, since we aren't using memcmp anymore.

Reply:
Changed to "==" operator and explicitly implemented the "!=" operator to use the negate